src/freenet/client/filter/TheoraBitstreamFilter.java

```diff
@@ -1,13 +1,14 @@
 package freenet.client.filter;
 
+import freenet.support.Logger;
+
 import java.io.IOException;
 import java.util.ArrayList;
-
-import freenet.support.Logger;
-import freenet.support.Logger.LogLevel;
+import java.util.List;
 
 public class TheoraBitstreamFilter extends OggBitstreamFilter {
```

Comment:
I would also totally love some tests for this class.

Reply:
I need some video for tests. What about this one SSK@gwe30-18LwLnAUuQqwZvxSeRk70LmxzGzYQtrnifknY,Nrw4BGOZ7H4hGyfCUVkZGuK9tLp9T6G7odPXOU5aj~Y,AQACAAE/small.ogv

Reply:
As soon as you can integrate some of the videos Arne linked in the other comment, I think this should be good to go.